src/templates/content.scss

```diff
@@ -18,6 +24,15 @@ $prefixPageContent: page--content;
     }
 }
 
+@media only screen and (min-width: 508px) {
```

Comment:
NAB: Should we get the standardized breakpoints mixin setup in React CM UI as well?

Reply:
I do think the mixins should come from this package, but for now I think we are okay, right?

Reply:
Right you are; hence NAB :-)

Reply:
Yep, yep. Thanks!